### PR TITLE
[ANCHOR-357] SEP-6 documentation updates

### PR DESCRIPTION
**Pull request by philipliu**
This adds SEP-6 support to the Anchor Platform API documentation.

Note: This will be merged when Anchor Platform 2.4 is released.

### CODE REVIEW

File: static/assets/rpc-methods/notify_refund_pending.json [hunk 1]
```diff
@@ -94,6 +97,7 @@
           "type": "integer",
           "description": "Protocol of the transaction",
           "enum": [
+            "6",
```
Comment (Ifropc) [inline static/assets/rpc-methods/notify_refund_pending.json:100]: Somehow it's broken in the preview, doesn't show SEP-31 badge for some reason, can you try re-generate mdx?

**Review by Ifropc — APPROVED** [review]

File: static/assets/rpc-methods/notify_refund_pending.json (hunk 1 above)
Comment (philipliu) [reply static/assets/rpc-methods/notify_refund_pending.json:100]: Does the RPC support SEP-31? It doesn't look like it according to the [handler](https://github.com/stellar/java-stellar-anchor-sdk/blob/c4e756846970136b267cd52c4891e16d1c799a4e/platform/src/main/java/org/stellar/anchor/platform/rpc/NotifyRefundPendingHandler.java#L96-L118).

Comment (Ifropc) [reply static/assets/rpc-methods/notify_refund_pending.json:100]: It's in the enum though, should it be removed?

Comment (philipliu) [reply static/assets/rpc-methods/notify_refund_pending.json:100]: The transaction object schema seems to be exactly the same across all RPC actions, regardless if they apply to the SEP. I just added it here for consistency.

Comment (Ifropc) [reply static/assets/rpc-methods/notify_refund_pending.json:100]: Ah nvm I was looking at the wrong place. I think tag should be `SEP - 6` not `SEP -6` in order for it to render correctly?
Also I'd prefer if SEP tags didn't have space between the dash, but it's something we can fix in another PR

Comment (philipliu) [reply static/assets/rpc-methods/notify_refund_pending.json:100]: > Ah nvm I was looking at the wrong place. I think tag should be SEP - 6 not SEP -6 in order for it to render correctly?

Nice catch, let me fix that.

> Also I'd prefer if SEP tags didn't have space between the dash, but it's something we can fix in another PR

I'll submit a PR to fix that after this is merged.

**Review by Ifropc — APPROVED** [review]

**Pull request merged**